### ISSUE TITLE
jsinspect does not support anonymous ES6 class

### ISSUE THREAD

**Issue by jbe456** [danielstjules/jsinspect#28]
jsinspect fails when code is using anonymous ES6 class: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/class

To reproduce:

1. Create a file with the following content:
```
var Foo = class {
  constructor() {}
  bar() {
    return "Hello World!";
  }
};

var instance = new Foo();
instance.bar(); // "Hello World!"
Foo.name;
```

2. run jsinspect & the following error is printed
```
[TypeError: Cannot read property 'type' of null]
```

**Comment by danielstjules** [comment]
Yup, sorry. I need to update https://github.com/danielstjules/acorn-bfs to work with the latest acorn and supported features, then will bump acorn versions. Thanks!

**Comment by danielstjules** [comment]
Fixed in 0.7.2, thanks!
